src/views/Settings.vue

```diff
@@ -54,70 +55,96 @@
+                :warning="inputErrors.includes('network')"
               >
-                <div class="text-gray mr-2">{{ $t('network') }}</div>
-                <div class="flex-auto">
+                <template v-slot:selected>
                   {{
                     form.network
                       ? networks[form.network].name
                       : $t('selectNetwork')
                   }}
-                </div>
-              </UiButton>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">{{ $t('settings.symbol') }}</div>
-                <input v-model="form.symbol" class="input flex-auto" required />
-              </UiButton>
-              <UiButton
+                </template>
+                <template v-slot:label>
+                  {{ $t(`settings.network`) }} *
+                </template>
+              </UiInput>
+              <UiInput
+                v-model="form.symbol"
+                placeholder="e.g. BAL"
+                :warning="inputErrors.includes('symbol')"
+              >
+                <template v-slot:label>
+                  {{ $t(`settings.symbol`) }} *
+                </template>
+              </UiInput>
+              <UiInput
                 @click="modalSkinsOpen = true"
-                class="text-left width-full mb-2 d-flex px-3"
+                :warning="inputErrors.includes('skin')"
               >
-                <div class="text-gray mr-2">{{ $t('settings.skin') }}</div>
-                <div class="flex-auto">
+                <template v-slot:selected>
                   {{ form.skin ? form.skin : $t('defaultSkin') }}
-                </div>
-              </UiButton>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">
-                  <Icon name="twitter" class="mr-1" />
-                </div>
-                <input v-model="form.twitter" class="input flex-auto" />
-              </UiButton>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">
-                  <Icon name="github" class="mr-1" />
-                </div>
-                <input v-model="form.github" class="input flex-auto" />
-              </UiButton>
-              <UiButton class="text-left width-full mb-2 d-flex px-3">
-                <div class="text-gray mr-2">{{ $t('settings.domain') }}</div>
-                <input v-model="form.domain" class="input flex-auto" />
-                <a
-                  class="d-block py-1 mr-n2"
-                  target="_blank"
-                  href="https://docs.snapshot.org/spaces/add-custom-domain"
-                >
-                  <Icon name="info" size="24" class="text-gray p-1" />
-                </a>
-              </UiButton>
+                </template>
+                <template v-slot:label>
+                  {{ $t(`settings.skin`) }}
```

Comment:
This is adding extra space between label and input, we dont want this extra space (same thing happen to others fields).

Before
![image](https://user-images.githubusercontent.com/16245250/114681826-b710e800-9d38-11eb-96c5-77e99652bb5f.png)
After
![image](https://user-images.githubusercontent.com/16245250/114681843-bc6e3280-9d38-11eb-88aa-8457a35b4ef4.png)

Reply:
I changed this back to `mr-2`: `<div class="text-gray mr-3">`